app/app/settings.py

```diff
@@ -74,12 +74,21 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.RemoteUserMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'applicationinsights.django.ApplicationInsightsMiddleware',
+    'custom_auth.CustomSSOMiddleware', 
 ]
 
+# added for custom SSO 
+CREATE_USER = env.bool("CREATE_USER", default=True)
+IS_SUPER = env.bool("IS_SUPER", default=False)
+HEADER_AUTH_PROXY_USER='HTTP_X_AUTH_PROXY_USER'
+HEADER_AUTH_PROXY_GROUPS='HTTP_X_AUTH_PROXY_GROUPS' # Group names are comma sperated e.g Admin,Guest
```

Comment:
Consider making the group separator customizable via an environment variable too.